flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -89,7 +89,7 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
         return "A320_Neo_CDU";
     }
     get isInteractive() {
-        return NXDataStore.get("MCDU_KB_INPUT", "DISABLED") === "ENABLED";
+        return true;
     }
```

Comment:
**question:** can you explain why this is needed ?

Reply:
called by the sim to determine if the display is a touchscreen or not, just return true always as this is more simple and doesn't require a reload to work.